examples/clinical_pipeline/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"index.mapping.coerce": false
```

Comment:
is this setting used anywhere?

Reply:
Not anymore, we can remove the file itself @Leolty